docs-en/chapter_stack_and_queue/queue.md

```diff
@@ -316,9 +316,9 @@ We can directly use the ready-made queue classes in programming languages:
 
 To implement a queue, we need a data structure that allows adding elements at one end and removing them at the other. Both linked lists and arrays meet this requirement.
 
-### Implementation Based on Linked List
+### Implementation Based on a Linked List
 
-As shown in the figure below, we can consider the "head node" and "tail node" of a linked list as the "head" and "tail" of the queue, respectively. We restrict the operations so that nodes can only be added at the tail and removed at the head.
+As shown in the figure below, we can consider the "head" and "tail" of a linked list as the "front" and "rear" of the queue, respectively. It is stipulated that nodes can only be added at the rear and removed at the front.
```

Comment:
Previous paragraph uses "element", we should replace "nodes" with "elements" to keep it consistent. And "specified" might be better than "stipulated".

Reply:
The `node` corresponds to the head node and tail node of the linked list, so i would recommend keeping it.